pkg/solana/cmd/chainlink-solana/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-plugin"
+	"github.com/pelletier/go-toml/v2"
+
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop"
+	"github.com/smartcontractkit/chainlink-solana/pkg/solana"
+)
+
+const (
+	loggerName = "PluginSolana"
+)
+
+func main() {
+	s := loop.MustNewStartedServer(loggerName)
```

Comment:
Ah right we might need to audit these since switching to one chain per relayer, and move chain id higher up. Same issue on all chains.

Reply:
Yeah, at some point I took care to name the logger uniquely when creating a relayer in core, not sure if that is still there and gives and ideas

Reply:
So in theory we would just have to change `<family>.<id>` to `<family>-<id>`, but it's not that simple because we don't have chain id yet when we boot up - maybe we should inject it via env var?

On the other hand, it would be nice to not have to rely on the plugin to do things correctly, and we are already piping the logs back through the host anyways. Instead of just passing them through named as-is, we could set the relay ID prefix on the host side.
https://github.com/smartcontractkit/chainlink-relay/blob/4c281ae24783040330c1ef13a609d2090ac300b1/pkg/loop/logger.go#L63

Reply:
Actually, I think we are already prefixing on the host side.
I wrote this up before realizing that: https://smartcontract-it.atlassian.net/browse/BCF-2709
Well we can double check and lock it in with a regression test anyways :shrug: